src/Modules/StatCan.OrchardCore.SurveyJS/Assets/surveyjs.js

```diff
@@ -0,0 +1,40 @@
+function initAssessment(surveyjs) {
+  var json = surveyjs.dataset.data
+  var id = surveyjs.dataset.id
+
+    Survey
+        .StylesManager
+        .applyTheme("default");
+
+    Survey
+        .JsonObject
+        .metaData
+        .addProperty("itemvalue", {name: "score:number"});
+    
+    window.survey = new Survey.Model(json);
+  
+    survey
+        .onComplete
+        .add(function (result) {
+
+          var plainData = result.getPlainData({
+            calculations: [{ propertyName: "score"}]
+          });
+          var totalScore = 0
+          function calcScore(data) {
+            return (data || [] ).reduce(function(sum, item) {
+              return sum+ (item.isNode ? calcScore(item.data) : item.score);
+              }, 0)
+          }
+          
+          totalScore = calcScore(plainData);
+
+          document
+            .querySelector('.surveyResult')
```

Comment:
In the case of multiple widgets on the same page, this would populate all of the results. you should reuse the surveyjs element above ?